test/performance/prival.cpp

```diff
@@ -44,4 +45,25 @@ static void PrivalFromString(benchmark::State& state) {
   SET_STATE_COUNTERS( state, prival_from_string );
 }
 
+static void GetPrivalString(benchmark::State& state) {
+  std::vector<int> prival_list =
+      { STUMPLESS_SEVERITY_ALERT | STUMPLESS_FACILITY_USER,
+        STUMPLESS_SEVERITY_CRIT | STUMPLESS_FACILITY_MAIL, 
+        STUMPLESS_SEVERITY_ERR | STUMPLESS_FACILITY_AUTH2,
+        STUMPLESS_SEVERITY_WARNING | STUMPLESS_FACILITY_FTP,
+        STUMPLESS_SEVERITY_NOTICE | STUMPLESS_FACILITY_NTP };
+
+  const char *result;
+  INIT_MEMORY_COUNTER( get_prival_string );
+
+  for (auto _ : state) {
+    for (auto prival: prival_list ) {
+      result = stumpless_get_prival_string( prival );
+    }
+  }
+  free( ( void * ) result );
```

Comment:
This call to `free` will need to move to inside both loops to avoid a memory leak in this test.